library/hawkular_alerts_group.py

```diff
@@ -0,0 +1,183 @@
+        """
+        try:
+            self.client.get_trigger(group_id)
+            return dict(
+                msg="Group trigger {group_id} already exist".format(group_id=group_id),
+                changed=self.changed)
+        except urllib2.HTTPError as err:
+            if err.code == 404:
+                try:
+                    #  create trigger object
+                    trigger = hawkular.alerts.Trigger()
+                    trigger.id = group_id
+                    trigger.name = name
+                    trigger.severity = getattr(hawkular.alerts.Severity, severity.upper())
+                    trigger.enabled = enabled
+
+                    self.client.create_group_trigger(trigger)
+                    self.changed = True
+                    return dict(
+                        msg="Successfully created group trigger {group_id}".format(group_id=group_id),
+                        changed=self.changed)
+                except Exception as e:
+                    self.module.fail_json(msg="Failed to create group trigger. Error: {error}".format(error=e))
+            else:
+                raise
+
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(
+            hawkular_api_hostname=dict(
+                default=os.environ.get('HAWKULAR_HOSTNAME'), type='str'),
+            hawkular_api_port=dict(
+                default=os.environ.get('HAWKULAR_PORT'), type='int'),
+            hawkular_api_auth_token=dict(
+                default=os.environ.get('HAWKULAR_TOKEN'), type='str'),
+            tenant=dict(required=True, type='str'),
+            name=dict(type='str'),
+            group_id=dict(required=True, type='str'),
+            severity=dict(type='str'),
+            state=dict(required=True, type='str', choices=['present', 'absent']),
+            scheme=dict(required=False, type='str', choices=['https', 'http'], default='https'),
+            enabled=dict(required=False, type='bool', default=True),
+        ),
+        required_if=[
+            ('state', 'present', ['name', 'severity'])
+        ],
+    )
+
+    for arg in ['hawkular_api_hostname', 'hawkular_api_port', 'hawkular_api_auth_token']:
+        if module.params[arg] in (None, ''):
+            module.fail_json(msg="missing required argument: {}".format(arg))
+
+    hostname = module.params['hawkular_api_hostname']
+    port     = module.params['hawkular_api_port']
+    token    = module.params['hawkular_api_auth_token']
+    tenant   = module.params['tenant']
+    name     = module.params['name']
+    group_id = module.params['group_id']
+    severity = module.params['severity']
+    scheme   = module.params['scheme']
+    state    = module.params['state']
+    enabled  = module.params['enabled']
+
+    # TODO: remove once the ssl cert verification is checked
+    import ssl
+    context = ssl._create_unverified_context()
+
+    hawkular_alerts = HawkularAlertsGroupTrigger(module, tenant, hostname, port, scheme, token, context)
```

Comment:
@simon3z I can do that but I tried to maintain consistency with all the previous modules